cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -627,6 +627,10 @@ describe('Dashboards', () => {
     cy.getByTestID('selector-list p').click()
     cy.getByTestID('time-machine-submit-button').click()
     cy.getByTestID('save-cell--button').click()
+
+    // freeze the clock to a date so we can come back and check screenshots
+    // relative to the same timestamp.
+    cy.clock(newDate)
```

Comment:
https://przemuh.pl/en/blog/stop-the-time-with-cyclock

Reply:
The problem was that the test was going back in time to compare the two graphs, but due to render times etc, the past time is not the same as the current. Therefore, the two snapshots won't be the same either.